SU-SOE/stanford_soe_helper
# SOE-2694: add text-decoration: underline;

# READY FOR REVIEW

# Summary
- Add underlines to the titles only news views.


# Needed By (Date)
- Sprint end

# Criticality
- Fixes broken stuff


# Steps to Test
- Switch to this branch
- Verify you see underlines on hover

# Affects 
- SoE (School of Engineering website)

# Associated Issues and/or People
## Related JIRA ticket(s)
https://stanfordits.atlassian.net/browse/SOE-2694

## Related PRs

## More Information
Check it out on _Dev_ at https://sites.stanford.edu/jse-soe-dev/about!

Kerri has approved the screenshot, but still needs to approve it on _Dev_.
Bill, will you please review the code and merge pending Kerri's approval.

## Folks to notify
@kerri-augenstein


# See Also
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
